selfhosting.html

```diff
@@ -50,7 +52,7 @@ <h3>Installing Debian</h3>
 While installing, do not install any desktop environment, but do install the SSH server.
 </p>
 <h3>Port forwaring</h3>
-<p>Every time you are going to use a port, you need to forward it. For example, HTTP is port 80, HTTPS is 443, etc. You need to set this up on your router's NAT settings (sometimes just port forwarding, differs per router). These steps differ for each router. Refer to your routers manual. A simple command to see what your IP address is, is <code>ifconfig</code>. This shows a lot of network info, but it will also show your local IP address needed for port forwarding.
+<p>Every time you are going to use a port, you need to forward it. For example, HTTP is port 80, HTTPS is 443, etc. You need to set this up on your router's NAT settings (sometimes just port forwarding, differs per router). These steps differ for each router. Refer to your routers manual. A simple command to see what your servers IP address is, is to run <code>ifconfig</code> on your server. This shows a lot of network info, but it will also show your local IP address needed for port forwarding.

```

Comment:
Don't like how the first sentence is worded but that's negligible.

Reply:
I see what you mean, it should be fixed. I also fixed some other minor things.